### DISCUSSION TITLE
Aqara leak sensor OTA

### DISCUSSION THREAD

**Discussion by SaschaRH** [Koenkk/zigbee2mqtt#18983] [Devices]
Hi,

I'm just curious why theres no OTA update (not even the chance to check for new versions) for SJCGQ11LM?
https://www.zigbee2mqtt.io/devices/SJCGQ11LM.html

Production date is 2023 and the firmware is from 2020...

Thanks,
Sascha

**Comment by lux73** [comment]
not all Devices could be upgraded via OTA or a Firmware File is available for it - e.g. none of my WSDCGQ11LM have seen an OTA
